exonum/tests/explorer/main.rs

```diff
@@ -191,6 +191,7 @@ fn test_explorer_errors_in_block() {
                 "tx_hash": block.header.tx_hash,
                 "state_hash": block.header.state_hash,
                 "error_hash": block.header.error_hash,
+                "entries": block.header.entries,
```

Comment:
Nit: There should be at least one test explicitly specifying the format of entries.